test/StringDiagram.lean

```diff
@@ -0,0 +1,56 @@
+import Mathlib.Tactic.Widget.StringDiagram
+import ProofWidgets.Component.Panel.SelectionPanel
+import ProofWidgets.Component.Panel.GoalTypePanel
+
+/-! ## Example use of string diagram widgets -/
+
+section MonoidalCategory
+
+open ProofWidgets
+
+open CategoryTheory
+open scoped MonoidalCategory
+
+universe v u
+
+variable {C : Type u} [Category.{v} C] [MonoidalCategory C]
+
+lemma left_triangle {X Y : C} (η : 𝟙_ _ ⟶ X ⊗ Y) (ε : Y ⊗ X ⟶ 𝟙_ _) (w : False) :
+    η ▷ X ≫ (α_ _ _ _).hom ≫ X ◁ ε = (λ_ _).hom ≫ (ρ_ _).inv := by
+  with_panel_widgets [SelectionPanel]
```

Comment:
It's okay to have tests that use it, but afaict it's ultimately a bit awkward to use `SelectionPanel` to show the goal type because it requires the extra click. `GoalTypePanel` is preferrable.